zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/X509AuthenticationUtil.java

```diff
@@ -103,14 +103,14 @@ public static X509TrustManager createTrustManager(ZKConfig config) {
 
       X509TrustManager tm = null;
       if (trustStoreLocation.isEmpty()) {
-        LOG.warn("X509AuthenticationUtil::Truststore not specified for client connection");
+        LOG.warn("Truststore location not specified for SSL");
       } else {
         try {
           tm = X509Util
               .createTrustManager(trustStoreLocation, trustStorePassword, trustStoreTypeProp,
                   crlEnabled, ocspEnabled, hostnameVerificationEnabled, false);
         } catch (X509Exception.TrustManagerException e) {
-          LOG.error("X509AuthenticationUtil::Failed to create trust manager", e);
+          LOG.error("Failed to create trust manager", e);
```

Comment:
neat: "Failed to create trust manager :"

Reply:
The standard/convention is to **not** add ":" at the end of such logging statements.

Examples: 
- https://github.com/linkedin/zookeeper/blob/ade1f2686a9740f206c1bc7e33a791a6b6416400/zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java#L523
- https://github.com/linkedin/zookeeper/blob/li-dev/base-3.6.3-znode-group-security/zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java#L850
- https://github.com/linkedin/zookeeper/blob/li-dev/base-3.6.3-znode-group-security/zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java#L874